docs/source/openshift_sdn/README.md

```diff
@@ -0,0 +1,66 @@
+This is openshift-sdn, the default network plugin for OpenShift (both
+OKD and OCP). It uses Open vSwitch to connect pods locally, with VXLAN
+tunnels to connect different nodes.
+
+OpenShift SDN is designed to be installed by the [OpenShift Network
+Operator](https://github.com/openshift/cluster-network-operator), and
```

Comment:
I think everything after the comma should be removed.

Reply:
This content is just migrated, please don't go over it yet.